token-lending/program/tests/helpers/mod.rs

```diff
@@ -943,7 +1003,7 @@ impl TestObligation {
             &[
                 create_account(
                     &payer.pubkey(),
-                    &obligation_keypair.pubkey(),
+                    &obligation.keypair.pubkey(),
```

Comment:
Had to change this because of lifetime stuff. 
obligation_keypair is dead, but obligation.keypair is still alive, so we loan it to the create_account function. 

Or something like that.

Reply:
isn't this like you are reusing stuff between tests rather than having it sandboxed?

Reply:
Hm, no I don't think this will cause it to be reused between tests. What do you mean?